src/EnergyPlus/Pumps.hh

```diff
@@ -292,8 +245,8 @@ struct PumpsData : BaseGlobalStruct
     Real64 PumpHeattoFluid = 0.0;  // Pump Power dissipated in fluid stream
     Real64 Power = 0.0;            // Pump Electric power
     Real64 ShaftPower = 0.0;       // Power passing through pump shaft
```

Comment:
This data structure needs to be cleaned out.  There is no reason inside EnergyPlus that we need a global-ish variable to hold `Power`.  Each pump can hold its own value of power consumption.  Whenever this change is made, it needs to be made cautiously because it can easily lead to diffs.